src/common/focus/focus.module.ts

```diff
@@ -1,18 +1,15 @@
 import {NgModule} from '@angular/core';
 import {FocusDirective} from './focus.directive';
 import {AutoFocusDirective} from './autofocus.directive';
-import {BlurDirective} from './blur.directive';
```

Comment:
blurdirective should be in this module

Reply:
Done in the commit: [Merge from Dev: conflicts resolving](https://github.com/qgrid/ng2/pull/39/commits/da05f15a228a611a5059e3049c9f18402446fa8f)